pkgs/development/tools/faq/default.nix

```
@@ -0,0 +1,47 @@
+{ buildGoModule, pkgs, lib, fetchFromGitHub }:
```

Also, it's common to use 1 line per argument:

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
}:
```

There's also some significance to the order of the arguments. Many people follow the rule of: "arguments should be ordered in the order of their usage in the expression, with the exception of `lib` which always comes 1st."